moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py

```diff
@@ -293,17 +293,27 @@ def update_ikfast_package(args):
   # Create a script for easily updating the plugin in the future in case the plugin needs to be updated
   easy_script_file_path = args.ikfast_plugin_pkg_path + "/update_ikfast_plugin.sh"
   with open(easy_script_file_path, 'w') as f:
-    f.write("rosrun moveit_kinematics create_ikfast_moveit_plugin.py" +
-            " --search_mode=" + args.search_mode +
-            " --srdf_filename=" + args.srdf_filename +
-            " --robot_name_in_srdf=" + args.robot_name_in_srdf +
-            " --moveit_config_pkg=" + args.moveit_config_pkg +
-            " " + args.robot_name +
-            " " + args.planning_group_name +
-            " " + args.ikfast_plugin_pkg +
-            " " + args.base_link_name +
-            " " + args.eef_link_name +
-            " " + args.ikfast_output_path)
+    f.write("search_mode="+args.search_mode+
+            "\nsrdf_filename=" + args.srdf_filename +
```

Comment:
What's the value of listing those parameters before calling `ros_run`? 
Most of them are used in conjuction with a verbosely named parameter, aren't they?

Reply:
this way you can look at the file and know what the values are, before calling it. I just prefer to know what the values are, before calling the script

Reply:
I've never used ikfast. Can you give an example of how the generated script would look before and after this change?